cluster/gce/gci/configure.sh

```diff
@@ -116,17 +116,17 @@ function split-commas {
 
 function install-gci-mounter-tools {
     CONTAINERIZED_MOUNTER_HOME="${KUBE_HOME}/containerized_mounter"
-    mkdir "${CONTAINERIZED_MOUNTER_HOME}"
+    mkdir -p "${CONTAINERIZED_MOUNTER_HOME}"
```

Comment:
Since you're already touching this code, can you please change the indentation in this function to be equal to 2 spaces (instead of 4 - for consistence with the whole rest of the file)?